packages/editor/src/plugins/rows/components/row-editor.tsx

```diff
@@ -36,8 +37,9 @@ export function RowEditor({
   }
 
   return (
-    <div
-      key={row.id}
+    <motion.div
+      layout
+      transition={{ duration: 0.3, type: 'linear' }}
```

Comment:
Maybe ask Gregor for transition type and duration in terms of good UX.

Reply:
Gregor saw the demo while the PR was a draft already 😄 

And the desired animation is shown in [the ticket](https://linear.app/serlo/issue/EDTR-169/moving-animation-when-switching-with-arrows) already. Would you say the end result doesn't match the video from the ticket?